python/pytests/golden/else_test/test_else_unordered_record.jsonl

```diff
@@ -0,0 +1,4 @@
+{"_time":"1970-01-01T00:00:00.000000001","_subsort":0,"_key_hash":17095134351192101601,"_key":"dev","joined":{"text":"Thread 1","user":"UCZ4","time":1,"thread_ts":1.0,"key":"dev"},"threads":{"text":"Thread 1","user":"UCZ4","time":1.0,"thread_ts":1.0,"key":"dev"},"non_threads":null}
```

Comment:
Can you elaborate on what is the expected results here? These appear to be correct. If `threads` is null then `non_threads` is used as the value of `joined`.

Reply:
the expected results are correct.  the test currently fails.  https://github.com/kaskada-ai/kaskada/actions/runs/5926509057/job/16068060314?pr=682#step:6:211

Reply:
i made you a failing test.